ctms/sync.py

```diff
@@ -129,6 +140,13 @@ def sync_records(self, db, end_time=None) -> Dict[str, Union[int, str]]:
         )
         main_fields = {entry.field for entry in main_acoustic_fields}
 
+        newsletters_mapping_entries: List[AcousticNewsletterMapping] = db.query(
+            AcousticNewsletterMapping
+        ).all()
+        newsletters_mapping = {
+            entry.source: entry.destination for entry in newsletters_mapping_entries
+        }
```

Comment:
Is this something we should define in `ctms/crud.py` as like `fetch_newsletter_mappings` or something? The idea being we try to contain as much of our database interaction as possible in that module.